### PR TITLE
Add Queries DisplayDetailsPpm and DisplayDetailsGl

### PR DESCRIPTION
**Pull request by sprucely**
Also tweaked tests to eliminate warnings and adjusted default location berry assets are stored to avoid write permission issues.

### CODE REVIEW

File: src/AggieEnterpriseApi/AggieEnterpriseApi.csproj [hunk 1]
```diff
@@ -7,7 +7,8 @@
     <Company>UC Davis</Company>
     <Description>Aggie Enterprise API</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
-	<LangVersion>latest</LangVersion>
+	  <LangVersion>latest</LangVersion>
+    <StrawberryShake_State>$(MSBuildProjectExtensionsPath)berry</StrawberryShake_State>
```
Comment (jSylvestre) [inline src/AggieEnterpriseApi/AggieEnterpriseApi.csproj:11]: Why is this added?

Comment (sprucely) [reply src/AggieEnterpriseApi/AggieEnterpriseApi.csproj:11]: Stores berry assets in `obj/berry/` instead of the default location, which is something like `C:\Program Files\dotnet\obj\berry`

Comment (sprucely) [reply src/AggieEnterpriseApi/AggieEnterpriseApi.csproj:11]: It's a better/more portable solution than requiring write access to a berry folder under program files.

File: src/AggieEnterpriseApi/Queries/DisplayDetailsGl.graphql [hunk 2]
```diff
@@ -0,0 +1,51 @@
+query DisplayDetailsGl($segmentString: GlSegmentString!, $validateCVRs: Boolean = true, $entity: String!,
```
Comment (jSylvestre) [inline src/AggieEnterpriseApi/Queries/DisplayDetailsGl.graphql:1]: @sprucely For the GL, you also want to get the details for Project, Program, and activity
(These will often just be default, but not always)

**Review by jSylvestre — CHANGES_REQUESTED** [review]

**Review by jSylvestre — APPROVED** [review]
